### PR TITLE
Seldon Serving: update headings, text, list numbering

### PR DESCRIPTION
**Pull request by kbhawkey**
Updated Seldon Serving page for Kubeflow Doc sprint.
I fiddled with the list item text (gateway, namespace label).
Happy to modify or change the text back to the original text.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1631)

### CODE REVIEW

**Comment by kbhawkey** [comment]
See issue #1615

File: content/docs/components/serving/seldon.md [hunk 1]
```diff
@@ -6,22 +6,20 @@ weight = 40
...
 
 You need to ensure the namespace where your models will be served has:
 
- 1. A Istio gateway named kubeflow-gateway
- 2. The namespace is labeled with `serving.kubeflow.org/inferenceservice=enabled`
+* An Istio gateway named kubeflow-gateway
+* A label set as `serving.kubeflow.org/inferenceservice=enabled`
 
-Examples:
+Example:
```
Comment (sarahmaddox) [inline content/docs/components/serving/seldon.md]: How about rewording this and the next line, to avoid the double set of phrases ending in colons. Something like this:

```
The following example applies the label `my-namespace` to the namespace for serving:
```

File: content/docs/components/serving/seldon.md [hunk 2]
```diff
@@ -49,12 +47,9 @@ spec:
...
 Save the above resource and apply it with `kubectl`.
- 
 
-### Examples
+## Examples
 
-   * [Kubeflow Seldon E2E Pipeline](https://docs.seldon.io/projects/seldon-core/en/latest/examples/kubeflow_seldon_e2e_pipeline.html)
+* [Kubeflow Seldon E2E Pipeline](https://docs.seldon.io/projects/seldon-core/en/latest/examples/kubeflow_seldon_e2e_pipeline.html)
 
 Seldon provides a [large set of example notebooks](https://docs.seldon.io/projects/seldon-core/en/latest/examples/notebooks.html) showing how to run inference code for a wide range of machine learning toolkits.
```
Comment (sarahmaddox) [inline content/docs/components/serving/seldon.md]: Looking at the "Examples" section, I'm now thinking it'd  be good to make the last item a bullet point too, to match the one on line 53. If we do that, then we need to change the item on line 53 to a full sentence. Suggestion for the item on line 53:

```
* The  [Kubeflow Seldon E2E Pipeline](https://docs.seldon.io/projects/seldon-core/en/latest/examples/kubeflow_seldon_e2e_pipeline.html) shows how to build re-usable components for an ML pipeline.
```

What do you think?

File: content/docs/components/serving/seldon.md (hunk 1 above)
Comment (kbhawkey) [reply content/docs/components/serving/seldon.md]: Good.

File: content/docs/components/serving/seldon.md (hunk 2 above)
Comment (kbhawkey) [reply content/docs/components/serving/seldon.md]: I completely agree. The one bullet point seemed somewhat odd.

Comment (kbhawkey) [reply content/docs/components/serving/seldon.md]: What do you think about no list for the `Examples`? Easy enough to put back.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-1631--competent-brattain-de2d6d.netlify.com/docs/components/serving/seldon/

File: content/docs/components/serving/seldon.md (hunk 2 above)
Comment (sarahmaddox) [reply content/docs/components/serving/seldon.md]: That looks good!

**Comment by sarahmaddox** [comment]
Thanks for the additional fixes!

/lgtm
/approve
